kevm-pyk/src/kevm_pyk/kproj/evm-semantics/evm.md

```diff
@@ -916,11 +916,19 @@ Some operators don't calculate anything, they just push the stack around a bit.
 These operations are getters/setters of the local execution memory.
 
 ```k
-    syntax UnStackOp ::= "MLOAD"
- // ----------------------------
+    syntax UnStackOp ::= "MLOAD" | "BLOBHASH"
+ // -----------------------------------------
     rule <k> MLOAD INDEX => #asWord(#range(LM, INDEX, 32)) ~> #push ... </k>
          <localMem> LM </localMem>
 
+   rule <k> BLOBHASH INDEX => 0 ~> #push ... </k>
```

Comment:
These rules seem fine but the placement of them, between MLOAD and MSTORE, seems off. I would place it next to BLOCKHASH, which it is much more similar to.